cmd/ipfs/dist/install.sh

```diff
@@ -1,21 +1,19 @@
 #!/bin/sh
 
 bin=ipfs
+binpaths="/usr/local/bin /usr/bin"
 
 # this script is currently brain dead.
 # it merely tries two locations.
 # in the future maybe use value of $PATH.
 
-binpath=/usr/local/bin
-if [ -d "$binpath" ]; then
-  mv "$bin" "$binpath/$bin"
-  echo "installed $binpath/$bin"
-  exit 0
-fi
+for binpath in $binpaths; do
+  if [ -d "$binpath" ]; then
```

Comment:
would be nice to also check `[ -w "$binpath" ]` to see if we're allowed to write in the given directory.